animations/test/misc/hero_animation_test.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/misc/hero_animation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createHeroAnimationDemoScreen() => MaterialApp(
+      home: HeroAnimationDemo(),
+    );
+
+void main() {
+  group('Hero Animation Tests', () {
+    testWidgets('Heigth of Container changes on Tap', (tester) async {
+      await tester.pumpWidget(createHeroAnimationDemoScreen());
+
+      // Get the initial Container.
+      final initialContainer =
+          tester.firstWidget(find.byType(Container)) as Container;
+
+      // Get height of Container.
+      // In this case minHeight is equal to maxHeight.
+
+      var initialHeight =
```

Comment:
nit: No need for this extra line gap between code & comments.

Reply:
Fixed this.